### PR TITLE
CNV-must-gather: Add E2E test to run on GCP

### PR DESCRIPTION
**Pull request by nunnatsa**
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

### CODE REVIEW

**Review by tiraboschi — APPROVED** [review]

File: core-services/image-mirroring/openshift/mapping_origin_4_8 [hunk 1]
```diff
@@ -129,7 +129,6 @@ registry.ci.openshift.org/origin/4.8:kubefed-operator quay.io/openshift/origin-k
 registry.ci.openshift.org/origin/4.8:kubemark-machine-controllers quay.io/openshift/origin-kubemark-machine-controllers:4.8 quay.io/openshift/origin-kubemark-machine-controllers:4.8.0
 registry.ci.openshift.org/origin/4.8:kubernetes-tests quay.io/openshift/origin-kubernetes-tests:4.8 quay.io/openshift/origin-kubernetes-tests:4.8.0
 registry.ci.openshift.org/origin/4.8:kubevirt-csi-driver quay.io/openshift/origin-kubevirt-csi-driver:4.8 quay.io/openshift/origin-kubevirt-csi-driver:4.8.0
-registry.ci.openshift.org/origin/4.8:kubevirt-must-gather quay.io/openshift/origin-kubevirt-must-gather:4.8 quay.io/openshift/origin-kubevirt-must-gather:4.8.0
```
Comment (bparees) [inline core-services/image-mirroring/openshift/mapping_origin_4_8:132]: why is this being removed?

Comment (nunnatsa) [reply core-services/image-mirroring/openshift/mapping_origin_4_8:132]: > why is this being removed?

As far as I understand it, this is not in use at all. @tiraboschi - WDYT?

Comment (bparees) [reply core-services/image-mirroring/openshift/mapping_origin_4_8:132]: well it's being published, it was published as recently as 37 minutes ago.  So it's hard to say "no one is using it".

assuming it's not fundamentally broken in some way, i don't see why we wouldn't continue to publish it.

that said, apparently we never published a 4.9/4.10/4.11 tag.

so maybe my real question should be, why aren't we publishing it generally?

(it looks like we published a 4.6 tag, and a 4.8 tag, only)

Comment (nunnatsa) [reply core-services/image-mirroring/openshift/mapping_origin_4_8:132]: Not sure about the history of it. But the formal release is done elsewhere. @tiraboschi - do you agree?

Comment (tiraboschi) [reply core-services/image-mirroring/openshift/mapping_origin_4_8:132]: yes, we are pushing to quay.io/repository/kubevirt/must-gather (without tagging),
I don't think we are using quay.io/openshift/origin-kubevirt-must-gather:4.8

**Comment by nunnatsa** [comment]
/cc @tiraboschi 

Could you please review again?

**Review by orenc1 — APPROVED** [review]
/approve

**Review by tiraboschi — APPROVED** [review]

**Pull request merged**